android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/view/ButtonProperties.kt

```diff
@@ -63,6 +63,7 @@ data class ButtonProperties(
       ServiceStatus.UPCOMING -> DefaultColor
       ServiceStatus.COMPLETED -> DefaultColor
       ServiceStatus.IN_PROGRESS -> WarningColor
+      ServiceStatus.EXPIRED -> DefaultColor
```

Comment:
Let's have an issue to review this. It seems that no color was selected for expired status based on this https://github.com/opensrp/fhircore/discussions/1557

Reply:
Yea good catch, that's odd